src/nvim/window.c

```diff
@@ -3877,10 +3876,12 @@ leave_tabpage (
 /*
  * Start using tab page "tp".
  * Only to be used after leave_tabpage() or freeing the current tab page.
- * Only trigger *Enter autocommands when trigger_enter_autocmds is TRUE.
- * Only trigger *Leave autocommands when trigger_leave_autocmds is TRUE.
+ * Only trigger *Enter autocommands when trigger_enter_autocmds is true.
```

Comment:
could be param style i e `@param trigger_enter_autocmds when true trigger *Leave autocmds`